src/data/roadmapData.json

```diff
@@ -238,14 +238,24 @@
       ]
     },
     {
-      "Html & css": [
+      "HTML & CSS": [
+        {
+          "name": "HTML",
+          "data": [
+            {
+              "description": "\"HTML (HyperText Markup Language) is the most basic building block of the Web. It defines the meaning and structure of web content. \n \n For example, content could be structured within a set of paragraphs, a list of bulleted points, or using images and data tables. \n \n HTML consists of a series of elements, which you use to enclose, or wrap, different parts of the content to make it appear a certain way, or act a certain way.\"",
+              "url": "https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics",
+              "urlLabel": "HTML basics"
+            }
+          ]
+        },
         {
-          "name": "Web elements",
```

Comment:
We got rid of this link, but it's kinda useful - it teaches about usage of selectors (that's loosely connected with CSS). What do you think about moving it somewhere?
Maybe whole section can be named HTML/CSS/Selectors, or there should be a new section?